### PR TITLE
Fix various unicode format errors

### PR DESCRIPTION
**Pull request by spikeliu**
Fix various unicode format problems, and add test cases for command with unicode as compound IDs

### CODE REVIEW

File: psamm/metabolicmodel.py [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 """Representation of metabolic network models."""
 
+from __future__ import unicode_literals
+
```
Comment (jonls) [inline psamm/metabolicmodel.py]: This module has a `__repr__` method on `FluxBounds` which needs to be corrected so that it returns `str` type in Python 2.7.

File: psamm/commands/excelexport.py [hunk 2]
```diff
@@ -93,16 +97,16 @@ def run(self):
...
+            media_sheet.write(x+1, 1, text_type(reaction))
+            media_sheet.write(x+1, 2, text_type(lower))
+            media_sheet.write(x+1, 3, text_type(upper))
 
         limits_sheet = workbook.add_worksheet(name='Limits')
 
         limits_sheet.write_string(0, 0, 'Reaction ID')
         limits_sheet.write_string(0, 1, 'Lower Limit')
-        limits_sheet.write_string(0, 2, str('Upper Limit'))
+        limits_sheet.write_string(0, 2, text_type('Upper Limit'))
```
Comment (jonls) [inline psamm/commands/excelexport.py]: The `text_type` call is not necessary here and can be removed completely.

File: psamm/metabolicmodel.py (hunk 1 above)
Comment (jonls) [inline psamm/metabolicmodel.py]: :+1:
